recipes/xsimd/all/conanfile.py

```diff
@@ -25,20 +25,19 @@ class XsimdConan(ConanFile):
     default_options = {
         "xtl_complex": False,
     }
-
     no_copy_source = True
 
     def requirements(self):
         if self.options.xtl_complex:
-            self.requires("xtl/0.7.4")
+            self.requires("xtl/0.7.5")
 
     def package_id(self):
         self.info.clear()
 
     def validate(self):
```

Comment:
now that you're doing minor improvements, reading https://github.com/xtensor-stack/xsimd#dependencies shows that if using xtl, the min cppstd is 14. Should we add that fact in the validate?

Reply:
I agree with you.
I'm goingo to try it.

Reply:
@RubenRBS 
I agree with you.
I am going to try it.
Thanks!

Reply:
fixed